frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -6,7 +6,6 @@
  * Copyright © 2020 Eric Herbrandson
  * Copyright © 2020 Kinvolk GmbH
  */
-
```

Comment:
Mistakenly removed?

Reply:
Not sure why the linter isn't handling these.